src/pyodide-py/pyodide/_core.py

```diff
@@ -16,5 +22,19 @@ class JsProxy:  # type: ignore
 
         # Defined in jsproxy.c
 
+        # Defined in jsproxy.c
+
+    def create_once_callback(obj: Callable) -> JsProxy:
+        """Wrap a Python callable in a Javascript function that can be called
+        once. After being called the proxy will decrement the reference count
+        of the Callable. The javascript function also has a `destroy` API that
+        can be used to release the proxy without calling it.
```

Comment:
I think you can remove the docstring short of a 1 line description. Otherwise we would have to keep it in sync. This function doesn't do this anyway.

Reply:
Well this is the version that will appear in the api docs.

Reply:
Right, forgot about that.